admin statistics view

rienafairefr opened the pull request:
last meeting, Nathalie asked me where the admin stats view went, and indeed the view is not there anymore. 
Of course, some stats can still be done through downloading the JSON from the API, but it's not a great UI.
I've built a small admin stats dashboard, using the `/statistics` endpoints, showing the number of users for each country and the number of experiments per month, using apexcharts.js integrated into vue, which I found easiest to integrate.

Code is fugly because I don't know javascript that well, but it seems to work. 

Pinging @vandaele because he was in that discussion

Commit:
apex charts stats view

Commit:
DRY, with a BarChartTable component

bformet reviewed (dismissed):
Looks pretty good at first glance. Though untested yet.
Re-adding stats is nice but its api relevance and performance should be discussed.
Copying what existed in old portal is perhaps not optimal.

Commit:
local storage in indexeddb instead of re-downloading

+ use SQL to query the data

Commit:
/statistics/nodes doesn't exist

Commit:
line-chart-table running count, remove alasql

Commit:
DRY ChartTable

Commit:
pills for filtering user creation year

Commit:
experimentsLoaded usersLoaded

Commit:
fix charttable

Commit:
fix running counts charts

Commit:
more charts

Commit:
more charts

Commit:
experimentsProgress

Commit:
fix getNodesStatistics

Commit:
more charts

Commit:
split into different sub views: experiments/users

Commit:
try to iterate experimentsStatistics only once

Commit:
fix tables, csv, etc

Commit:
migrate to chart.js (canvas)

Commit:
downsample a bit

Commit:
remove downsample, instead gruop to a point per day

Commit:
MAX_COUNTRY_ENTRIES, 'Others' category

Commit:
remove downsample, add zoom plugin

Commit:
chart download canvas as png

Commit:
show stat coming from network or storage

vandaele committed:
give full container width to pie chart

vandaele committed:
move the chart table under the chart

vandaele committed:
Use h3 heading for chart titles

vandaele committed:
Replace dropdown menu with buttons

Download buttons are directly available, instead of a 1-more click with
dropdown menu.
Putting it on the right with flex box instead of float, gives the full
container width for data table.

Commit:
fix granularity to day for charts over time

Commit:
fix missing value_title values

Commit:
better exportName

Commit:
Merge pull request #1 from vandaele/stats

Stats: chart table and download buttons rework

Commit:
fix stacked fill

Commit:
fix some titles

Commit:
no legend for line/vbar charts (only one dataset)

Commit:
fix axis title display for stacked relative area

Commit:
granularity one day for running count of experiments
